### ISSUE TITLE
use at a basic level not obvious

### ISSUE THREAD

**Issue by RobP54** [chrdavis/SmartRename#46]
just downloaded and installed, selected a number of files in explorer and initiated smartrename, I just want to rename them all to an arbitary value, why do I need to have file names in the "search" field to give any result, is there a wild card to enter that will select all files?

Its far from clear...

**Comment by LesFerch** [comment]
Presumably you wanted something like this:

New Name.txt
New Name (2).txt
New Name (3).txt

Explorer does that on its own. Just select the files and enter a new name. It will look like it's only going to change one file, but when you press `Enter`, all of the selected files will be renamed.

If you want to do that using SmartRename, check `Use regular expressions`, enter `.*` in the Search for field and the new name in the Replace field.

**Comment by ihorbond** [comment]
Agree, not intuitive at all. You would think that what you enter search is used to identify the files you want to update but no, it's more like search and replace the matching part unless you use wildcard regex.
